marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/DeployHubAmpsCommand.java

```diff
@@ -63,53 +53,7 @@ public DeployHubAmpsCommand(HubConfig hubConfig) {
      */
     @Override
     public void execute(CommandContext context) {
-
-        // this is a place to optimize -- is there a way to get
-        // server versions without an http call?
-        String serverVersion = versions.getMarkLogicVersion();
-
-
-        logger.info("Choosing amp installation based on server version " + serverVersion);
-
-        if (serverVersion.matches("^[789]\\.0-[1234]\\.\\d+")) {
-            throw new DataHubConfigurationException("DHF " + hubConfig.getDHFVersion() +" cannot deploy security to server version " + serverVersion);
-        }
-        if (serverVersion.startsWith("9.0-5")) {
-            logger.info("Using non-SSL-compatible method for 9.0-5 servers, for demos only");
-            String modulesDatabaseName = hubConfig.getAppConfig().getModulesDatabaseName();
-            ManageConfig manageConfig = context.getManageClient().getManageConfig();
-            String securityUsername = manageConfig.getSecurityUsername();
-            String securityPassword = manageConfig.getSecurityPassword();
-            DatabaseClient installerClient = DatabaseClientFactory.newClient(
-                hubConfig.getHost(),
-                8000,
-                "Security",
-                new DatabaseClientFactory.DigestAuthContext(securityUsername, securityPassword)
-            );
-            //new AmpsInstaller(securityStagingClient).installAmps(modulesDatabaseName);
-            ServerEvaluationCall call = installerClient.newServerEval();
-            try (InputStream is = new ClassPathResource("installer-util/install-amps.xqy").getInputStream()) {
-                String ampCall = IOUtils.toString(is, "utf-8");
-                is.close();
-                ampCall = ampCall.replaceAll("data-hub-MODULES", modulesDatabaseName);
-                call.xquery(ampCall);
-                call.eval();
-            } catch (IOException e) {
-                throw new DataHubConfigurationException(e);
-            }
-        } else {
-            logger.info("Using CMA for servers starting with 9.0-6");
-            String modulesDatabaseName = hubConfig.getAppConfig().getModulesDatabaseName();
-            ManageClient manageClient = context.getManageClient();
-
-            try (InputStream is = new ClassPathResource("hub-internal-config/configurations/amps.json").getInputStream()) {
-                String payload = IOUtils.toString(is, "utf-8");
-                payload = payload.replaceAll("data-hub-MODULES", modulesDatabaseName);
-                manageClient.postJsonAsSecurityUser("/manage/v3", payload);
-            } catch (IOException e) {
-                throw new DataHubConfigurationException(e);
-            }
-        }
+        super.execute(context);
```

Comment:
I think removing the amps creation using the install-amps.xqy would slow the amps installation in 9.0-5

Reply:
The ticket states "make sure to scaffold out the 65ish amps and use them to deploy (but not remove) for 9.05 (and previous) and still use cma for 9.06 and later." I believe the reasoning is due to not wanting to maintain multiple sets of amps (the scaffolded out and the condensed set) and additional code for loading amps. I may be misinterpreting though.

Reply:
@ryanjdew ,
The ticket does state to use the scaffolded amps for 9.0-5 installation and cma for 9.0-6 and above. I am guessing that the installation will be slow in 9.0-5 as a result. 
@aebadirad, is that ok ?

Reply:
Yeah, 9.05 folks get the slow file by file request. It was to reduce the keeping of multiple amps. This one is okay to keep sanity going forward.